src/platform-includes/performance/automatic-performance-metrics/android.mdx

```diff
@@ -2,3 +2,15 @@ If configured, the Android SDK automatically collects the following performance
 
 * <PlatformLink to="/performance/instrumentation/automatic-instrumentation/#app-start-instrumentation">Cold and warm app start time</PlatformLink>.
 * <PlatformLink to="/performance/instrumentation/automatic-instrumentation/#slow-and-frozen-frames">Slow and frozen frame rendering</PlatformLink>.
+
+### Application Not Responding (ANR) Rate
+
+<Include name="early-adopter-note.mdx" />
+
+If configured, the Android SDK reports [Application Not Responding (ANR)](platforms/android/configuration/app-not-respond/) errors if the main UI thread of the application is blocked for more than five seconds. ANR rate is the percentage of distinct users who have experienced any kind of ANR during the specified time period. Foreground ANR rate is the percentage of distinct users who have experienced an ANR when the app was running in the foreground. It is more likely that foreground ANRs were perceieved by the user.
+
```

Comment:
```suggestion
Application Not Responding (ANR) rate is defined as the percentage of unique users in a given time period, who’ve experienced the lack of response to an input event (such as a key press or a screen touch), in an app.

When configured, Sentry's Android SDK, version x.x and above, reports [Application Not Responding (ANR)](platforms/android/configuration/app-not-respond/) errors if the main UI thread of your application is blocked for more than five seconds. 
```
Two questions:
1. Do we know the version number yet?
2. The link appears to be broken and going to a page not found. I assume we're adding something here so it will be fixed?

Reply:
@lizokm the sdk version number applies to ANR rate - so I don't think it belongs here in the second paragraph where we explain ANR errors. ANR errors have been available for a while. However, the configuration I was trying to link to (it's missing a slash, hence the broken link) https://docs.sentry.io/platforms/android/configuration/app-not-respond/ applies to both anr and anr rate.

Reply:
> Application Not Responding (ANR) rate is defined as the percentage of unique users in a given time period, who’ve experienced the lack of response to an input event (such as a key press or a screen touch), in an app.

This also doesn't capture what an ANR is- "who’ve experienced the lack of response to an input event (such as a key press or a screen touch), in an app" is one type of ANR. Technically an ANR is when the main ui thread of the app is blocked for more than 5 seconds. It doesn't always have to be in response to an input event